packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -101,6 +101,20 @@ Request.prototype.type = function (type) {
 	return this;
 }
 
+Request.prototype.toJSON = function(){
+	return {
+		aborted: this.aborted,
+		agent: this._agent,
+		headers: this._headers,
+		method: this._method,
+		postParams: this._postParams,
+		queryParams: this._queryParams,
+		timeout: this._timeout,
+		type: this._type,
+		urlPath: this._urlPath,
```

Comment:
@gigabo It's possible :grin:. I don't recall anything though. There's some special handling of ReactServerAgent _responses_ in Cache.js (sibling to Request.js), and there's some special handing of error objects in Redfin-internal code, but I don't think I recall any Request-related handling.

@doug-wade there's also a `_cacheWhitelist` property (whose name doesn't make any sense unless you look at the implementation) that includes an array of headers that we're interested in from the response. It's technically state on the request; not sure if it's in interesting in a JSON representation of the request.

Somewhat related, are we sure we want to include the raw `agent` object in a `toJSON` method? If set, it's a node HTTP Agent, which doesn't really seem JSON-y to me (though maybe it has properties we want to expose?)